cmake/linux/package_linux.sh.in

```diff
@@ -6,11 +6,10 @@
 # Notes:    Will attempt to fetch linuxdeployqt automatically (x86_64 only)
 #           See Also: https://github.com/probonopd/linuxdeployqt/blob/master/BUILDING.md
 
-set -e
+trap "error 'Failed to generate AppImage'; exit 1" ERR
 
 LINUXDEPLOYQT="@CMAKE_BINARY_DIR@/linuxdeployqt"
 VERBOSITY=2 # 3=debug
-LOGFILE="@CMAKE_BINARY_DIR@/appimage.log"
```

Comment:
This was written to a log because the output from `linuxdeployqt` is pretty large.  Furthemore, it's nice to have access to review this for warnings and other things.  I'm not sure removing it unless non-zero error code is a good idea.  Can you please explain this reasoning?

Reply:
My reasoning was it runs on the CI and you never see it anyway unless it fails but I realize how stupid logic that is now when I say it. Will keep log file.